gamemodes/terrortown/gamemode/shared/sh_speed.lua

```diff
@@ -12,7 +10,9 @@ SPEED = SPEED or {}
 -- @internal
 -- @realm shared
 function SPEED:HandleSpeedCalculation(ply, moveData)
-	if not ply:IsTerror() then return end
+	if not ply:IsTerror() then
+		return
+	end
```

Comment:
```suggestion
	if not ply:IsTerror() then return end
```

I know you're using a formatter, but this breaks our code style